### DISCUSSION TITLE
Removing anything related to like or twitter and X

### DISCUSSION THREAD

**Discussion by mi8nd** [community/community#126686] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

[index.txt](https://github.com/user-attachments/files/15509978/index.txt)
this is my html code but I want to delete anything related to like or twitter and X

**Comment by ebndev** [comment]
Hi @mi8nd, please repost your discussion without a link that automatically downloads if you still need assistance.

Thank you!
